test/test_mediafile.py

```diff
@@ -449,7 +451,11 @@ def test_read_full(self):
     def test_read_empty(self):
         mediafile = self._mediafile_fixture('empty')
         for field in self.tag_fields:
-            self.assertIsNone(getattr(mediafile, field))
+            value = getattr(mediafile, field)
+            if isinstance(value, list):
+                assert len(value) == 0
```

Comment:
`not value` is a slightly more Pythonic way to write `len(value) == 0`.

Reply:
As I'm already checking a type - you are 100% right, I'll change it :D

Reply:
Fixed!